integration-test/1482-maritime_boundary-buffered_land.py

```diff
@@ -0,0 +1,53 @@
+from . import FixtureTest
+
+
+# Adds tests for OSM features (but not NE features)
+class MaritimeBoundary(FixtureTest):
+
+    def test_usa_canada_country_boundary(self):
+        # country boundary of USA and Canada
+        self.load_fixtures([
+            'https://www.openstreetmap.org/relation/148838',
+            'https://www.openstreetmap.org/relation/1428125',
+            'file://integration-test/fixtures/buffered_land/'
```

Comment:
By hosting the shapefile in a `buffered_land` directory FixtureTest will set that as the database table name on import.